server/api/room.js

```diff
@@ -13,13 +13,15 @@ router.get('/', async (req, res, next) => {
   }
 });
 
-// GET - /api/room/create/
+// POST - /api/room/
 // Makes a new room and returns it
 // Maybe change to POST if we want the firebase room name on the room
-router.post('/create/:roomId', async (req, res, next) => {
+router.post('/', async (req, res, next) => {
   try {
-    const { roomId } = req.params;
-    const room = await Room.create({ name: roomId });
+    const { roomName } = req.body;
```

Comment:
had to make this passed in via body because, if we put it in params, in the rare case that someone decided to name their room 'create' , we would be screwed, because express would not be able to tell which route to do, /api/room/create or /api/room/:paramater where paramater is create